manifests/osfamily/windows.pp

```diff
@@ -8,8 +8,12 @@
   } elsif  ($::puppet_agent::is_pe and (!$::puppet_agent::use_alternate_sources)) {
     $pe_server_version = pe_build_version()
     $tag = $::puppet_agent::arch ? {
-      'x64' => 'windows-x86_64',
-      'x86' => 'windows-i386',
+      'x64' => $::fips_enabled ? {
+          true => 'windowsfips-x64',
+          default => 'windows-x86_64' },
+      'x86' => $::fips_enabled ? {
+          true => 'windowsfips-i386',
+          default => 'windows-i386' },
```

Comment:
We should test that based `fips_enabled` flag the correct package is set in the manifest